includes/ProductSets/Sync.php

```diff
@@ -122,7 +140,7 @@ public function add_hooks() {
 		// product set hooks, compare taxonomies the lists before and after saving product to see if must sync
 		add_action( 'created_fb_product_set', array( $this, 'fb_product_set_hook_before' ), 1 );
 		add_action( 'created_fb_product_set', array( $this, 'fb_product_set_hook_after' ), 99 );
-		add_action( 'edited_fb_product_set', array( $this, 'fb_product_set_hook_before' ), 1 );
+		add_action( 'edit_fb_product_set', array( $this, 'fb_product_set_hook_before' ), 1 );
```

Comment:
Was this made on purpose, I mean the change to action name? Should we keep it `edited_fb_product_set`?

Reply:
Yes this is on purpose. `edit_fb_product_set` runs before and I allow to get the "before" term name and store it in the static property and compare it to the new name when `edited_fb_product_set` runs later.